spec/factories/consents.rb

```diff
@@ -47,7 +47,7 @@
     parent_name { Faker::Name.name }
     parent_email { Faker::Internet.email(domain: "gmail.com") }
     # Replace first two digits with 07 to make it a mobile number
-    parent_phone { Faker::PhoneNumber.cell_phone }
+    parent_phone { "07700 900#{random.rand(0..999).to_s.rjust(3, "0")}" }
```

Comment:
As it's used in a few places, is it worth pushing this into a spec helper method somewhere?

Reply:
Could also document why we've had to do this as a comment...

Reply:
Sure. I thought the same thing, I guess if we're both thinking it it must be a good idea.

Reply:
➕